intranetunits/models.py

```diff
@@ -71,19 +65,14 @@ class IntranetUnitsReportsPage(BasePage):
     ] + BasePage.content_panels
 
     search_fields = BasePage.search_fields + [
-        index.SearchField('get_report_summaries_for_indexing', partial_match=True),
-        index.SearchField('get_report_doc_links_for_indexing', partial_match=True),
+        index.
+        SearchField('get_report_summaries_for_indexing', partial_match=True),
+        index.
+        SearchField('get_report_doc_links_for_indexing', partial_match=True),
```

Comment:
I was wrong. My autofixer does this too now! It's annoying but I guess we have to live with it. No big deal.